angler/config/bytecode-proprietary-api24.txt

```diff
@@ -2,6 +2,8 @@
 
 system/app/HwMMITest/HwMMITest.apk
 system/app/HwSarControlService/HwSarControlService.apk
+system/app/SetupSmartDeviceOverlay/SetupSmartDeviceOverlay.apk
+system/app/Tycho/Tycho.apk
```

Comment:
One other thing that needs to be added that I found but did not submit is GCS.apk. That is Google Connectivity Services. That is also necessary for Project Fi out of the box to work.